presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestAggregations.java

```diff
@@ -216,7 +215,7 @@ public void testCountDistinct()
                         "  GROUP BY orderkey " +
                         "HAVING COUNT(DISTINCT partkey) != CARDINALITY(ARRAY_DISTINCT(ARRAY_AGG(partkey))))",
                 "VALUES 0");
-        assertQuery(builder(TEST_SESSION)
+        assertQuery(builder(getSession())
```

Comment:
Is this a valid change?

Reply:
Yes, individual runners should be running with their session defaults to ensure that runtime specific configs are available. Without this change, presto-on-spark code runs into error when it tries to access configs exclusively applicable to presto-on-spark.